aytests/set_passwords.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Check if all users has some value in the password field
+# (bsc#973639, bsc#974220, bsc#971804 and bsc#965852)
+
+set -e -x
+
+# It's OK if we DO NOT find a blank password.
+getent shadow | grep -P "^[^:]+::" || echo "AUTOYAST OK"
```

Comment:
`-P` for `--perl-regexp` seems too strong. `-E`, `--extended-regexp` works just as well.